libmscore/figuredbass.h

```diff
@@ -277,6 +278,8 @@ class FiguredBass : public Text {
       static bool       fontData(int nIdx, QString *pFamily, QString *pDisplayName,
                               qreal * pSize, qreal * pLineHeight);
 
+      virtual std::vector<FiguredBassItem*> getFiguredBaseItems() { return items; };
+
```

Comment:
Please not public. Or at least `const std::vector<const FiguredBassItem*>` (not sure I got the syntax right, but the gol is that neither the vector nor each of the items should be modifiable from outside).